website/templates/emails/notify_base.mako

```diff
@@ -43,7 +43,7 @@
                 <tbody>
                     <tr>
                         <td style="border-collapse: collapse;">
-                            <p class="small text-center" style="text-align: center;font-size: 12px;">To change your email notification preferences, visit your project or user settings.</p>
+                            <p class="small text-center" style="text-align: center;font-size: 12px;">To change your email notification preferences, visit your project or <a href="http://osf.io/settings">user settings</a>.</p>
```

Comment:
@sloria Please take a look at this, not sure if there is a better way to get what is wanted in https://openscience.atlassian.net/browse/OSF-4142?focusedCommentId=23888&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-23888 while still using the base template.